.travis.yml

```diff
@@ -5,9 +5,9 @@ python:
     - "3.3"
 
 install:
-    - pip install pytest-cov --use-mirrors
-    - pip install pytest-pep8 --use-mirrors
-    - pip install coveralls --use-mirrors
+    - pip install pytest-cov
+    - pip install pytest-pep8
+    - pip install coveralls
```

Comment:
I changed the CI build process in #79 so that TravisCI just runs Tox. Do you mind pulling in latest `master` and removing these changes to `.travis.yml`? Thanks!